frontend/public/components/resource-quota.jsx

```diff
@@ -29,7 +35,56 @@ const Details = ({obj: rq}) => <React.Fragment>
 </React.Fragment>;
 
 export const ResourceQuotasList = props => <List {...props} Header={Header} Row={Row} />;
-export const ResourceQuotasPage = props => <ListPage {...props} ListComponent={ResourceQuotasList} kind={kind} canCreate={true} />;
+
+export const quotaType = quota => {
+  if (!quota) {
+    return undefined;
+  }
+  return quota.metadata.namespace ? 'namespace' : 'cluster';
+};
+
+// Split each resource quota into one row per subject
+export const flatten = resources => _.flatMap(resources, resource => _.compact(resource.data));
+
+const ResourceQuotasPage_ = props => {
+  const {match: {params: {name, ns}}} = props;
+  const clusterSelected = ['cluster'];
+  const clusterItems = [{id: 'cluster', title: 'Cluster-wide Resource Quotas'}];
+
+  let resources = [{kind: 'ResourceQuota', namespaced: true}];
+  let selected = ['namespace'];
+  let items = [{id: 'namespace', title: 'Namespace Resource Quotas'}];
+
+  if (flagPending(props.flags[FLAGS.OPENSHIFT])) {
+    return <LoadingBox />;
+  }
+  if (props.flags[FLAGS.OPENSHIFT]) {
+    resources.push({kind: 'ClusterResourceQuota', namespaced: false, optional: true});
+    items = clusterItems.concat(items);
+    selected = clusterSelected.concat(selected);
+  }
+  return <MultiListPage
+    canCreate={true}
+    createButtonText="Create Resource Quota"
+    createProps={{to: `/k8s/ns/${ns}/resourcequotas/new`}}
+    ListComponent={ResourceQuotasList}
+    staticFilters={[{'resource-quota-roleRef': name}]}
+    resources={resources}
+    filterLabel="Resource Quotas by name"
+    label="Resource Quotas"
+    namespace={ns}
+    flatten={flatten}
+    title="Resource Quotas"
+    rowFilters={[{
```

Comment:
Row filters should only be supplied if this is OpenShift. Otherwise there's only one kind of quota and no need for the filter.